src/olympia/tags/migrations/0004_auto_20210721_1206.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.4 on 2021-07-21 12:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tags', '0003_auto_20210721_1146'),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name='tag',
+            name='tag_blacklisted_num_addons_idx',
+        ),
+        migrations.AlterField(
+            model_name='tag',
+            name='denied',
+            field=models.BooleanField(default=False, null=True),
+        ),
+        migrations.AlterField(
+            model_name='tag',
+            name='restricted',
+            field=models.BooleanField(default=False, null=True),
+        ),
```

Comment:
Do we have an issue filed to drop these columns in a later push?

Reply:
https://github.com/mozilla/addons-server/issues/17545